src/pip/_internal/commands/show.py

```diff
@@ -67,12 +67,21 @@ def search_packages_info(query):
     if missing:
         logger.warning('Package(s) not found: %s', ', '.join(missing))
 
+    def get_requiring_packages(package_name):
+        return [
+            pkg.project_name for pkg in pkg_resources.working_set
+            if canonicalize_name(package_name) in
```

Comment:
I would assign `canonicalize_name(package_name)` outside the list comprehension and then use the variable inside, which looks a little simpler (also avoids recomputing every time).